solidity/hardhat.config.ts

```diff
@@ -0,0 +1,61 @@
+import "@nomiclabs/hardhat-waffle";
+import "hardhat-gas-reporter";
+import "hardhat-typechain";
+import { task } from "hardhat/config";
+
+task("accounts", "Prints the list of accounts", async (args, hre) => {
+  const accounts = await hre.ethers.getSigners();
+
+  for (const account of accounts) {
+    console.log(account.address);
+  }
+});
+
+// // This is a sample Buidler task. To learn how to create your own go to
+// // https://buidler.dev/guides/create-task.html
+// task("accounts", "Prints the list of accounts", async (taskArgs, bre) => {
+//   const accounts = await bre.ethers.getSigners();
+
+//   for (const account of accounts) {
+//     console.log(await account.getAddress());
+//   }
+// });
+
+// You have to export an object to set up your config
+// This object can have the following optional entries:
+// defaultNetwork, networks, solc, and paths.
+// Go to https://buidler.dev/config/ to learn more
+module.exports = {
+  // This is a sample solc configuration that specifies which version of solc to use
+  solidity: {
+    version: "0.8.10",
+    settings: {
+      optimizer: {
+        enabled: true
+      }
+    }
+  },
+  networks: {
+    althea: {
+      url: "http://localhost:8545",
```

Comment:
All of the test accounts used in Gravity are removed here since hardhat can't control the EVM. The accounts here are the ones defined in genesis in the setup-validators.sh script.